kopuz/assets/tailwind.css

```
@@ -689,6 +751,10 @@
     --tw-translate-y: calc(calc(1/2 * 100%) * -1);
     translate: var(--tw-translate-x) var(--tw-translate-y);
   }
+  .translate-y-2 {
+    --tw-translate-y: calc(var(--spacing) * 2);
+    translate: var(--tw-translate-x) var(--tw-translate-y);
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)